sources/packages/web/src/components/aest/students/assessment/History.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-container>
+        <body-header title="History" class="m-1"> </body-header>
```

Comment:
Can we please add the _recordsCount_ also here?

Reply:
recount is not in figma, 
![image](https://user-images.githubusercontent.com/77353155/160471110-d18c5bcf-aeb5-44df-b9aa-e422e0740efe.png)
 i will double check with Lynn

Reply:
Please double check but there is no reason to not add them here 😉

Reply:
checked with @abschwenker and added the record count